tests/rms/test_copy_rms_param_to_ertbox_grid.py

```diff
@@ -0,0 +1,214 @@
+"""Run tests in RMS using rmsapi to test geogrid_to_ertbox_field_params.py
+
+Creates a tmp RMS project in given version.
+
+
+This requires a RMSAPI license, and to be ran in a "roxenvbash" environment
+
+"""
+
+import contextlib
+import shutil
+from os.path import isdir
+from pathlib import Path
+
+import numpy as np
+import pytest
+import xtgeo
+
+with contextlib.suppress(ImportError):
+    import rmsapi
+
+
+from aps.utils.constants.simple import Debug
```

Comment:
I think you'll need to put this aps import under the contextlib suppress context as well

Reply:
Will try.

Reply:
I tried without success, but I made the fmu-tools code independent of the APS code to avoid the problem and make too much dependencies back and forth between fmu-tools and APS-Facies repo